dataops/components/task_stream/crud.py

```diff
@@ -17,10 +17,12 @@
 import json
 from time import time
 
+from common import LoggerFactory
```

Comment:
Can we initialize LoggerFactory here instead of in `views.py` to ensure `__name__` is mapped to exact location the exception will be raised? It'll also prevent having to pass a logger instance to `get_status_with_sse`.

```suggestion
from common import LoggerFactory
settings = get_settings()
logger = LoggerFactory(
    __name__,
    level_default=settings.LOG_LEVEL_DEFAULT,
    level_file=settings.LOG_LEVEL_FILE,
    level_stdout=settings.LOG_LEVEL_STDOUT,
    level_stderr=settings.LOG_LEVEL_STDERR,
).get_logger()

class StreamCRUD(StreamParser, RedisCRUD):
...
```